### ISSUE TITLE
"options" is mandatory according to typescript

### ISSUE THREAD

**Issue by ValYouW** [dotcore64/react-stay-scrolled#130]
Hi,

In the typedef file the `options` parameter is mandatory:
```ts
export default function useStayScrolled(
    domRef: React.RefObject<HTMLElement>,
    options: UseStayScrolledOptions,
): UseStayScrolled;
```

Should be marked as optional:
`options?: UseStayScrolledOptions,`

or is it not meant to be optional??

**Comment by perrin4869** [comment]
Hey, thanks for the issue, you're right, let me fix this!

**Comment by perrin4869** [comment]
Fixed by 7.3.1
